app/services/feature_flag.rb

```diff
@@ -50,6 +60,17 @@ def self.sync_with_database(feature_name, active)
     feature.save!
   end
 
+  def self.sync_with_database_with_object(feature_name, object, active)
+    ActiveRecord::Base.transaction do
+      feature = Feature.find_or_create_by!(name: feature_name)
+      if active
+        feature.selected_objects.find_or_create_by!(object: object)
+      else
+        feature.selected_objects.where(object: object).destroy_all
+      end
+    end
+  end
```

Comment:
Right, so an existence of the selected object is equivalent with the flag being on for whatever it is pointing at?

Reply:
In the first iteration, yes. I'd guess later on we could put some more data on that (and make it for example an exception of otherwisee activated feature) but for now its presence is simply taken as activation.